.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+## Types of changes
+<!--
+  What types of changes does your code introduce? Put an `x` in all the boxes
+  that apply:
+-->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
```

Comment:
Would it make more sense to have separate bugfix and new feature PR templates?

Reply:
Possibly, though I'm not sure if you can do that